### ISSUE TITLE
Angle Degrees/Radians inconsistency

### ISSUE THREAD

**Issue by DeinFreund** [MauriceGit/Quaternion_Library#2]
`mtCreateMTQuaternion` requires the angle to be given in radians, but `mtAngleVectorVector` returns an angle in degrees. As these two libraries are intended to be used together it would be nice if they used the same units.

**Comment by MauriceGit** [comment]
Yes, you are right. I will change that some time the next days (also there is an unused and not provided include, that needs to be removed).
If you like, you can also provide the pull request.
